src/main/java/org/grizz/model/Building.java

```diff
@@ -0,0 +1,15 @@
+package org.grizz.model;
+
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@Builder
+@Document(collection = "buildings")
+public class Building {
+
+    private String type;
```

Comment:
How about changing this with enum type? `BuildingType.SOLAR_POWER_PLANT`, `BuildingType.METAL_MINE`, `BuildingType.CRYSTAL_MINE` etc